taskfiles/dev.yml

```diff
@@ -23,6 +23,17 @@ tasks:
     - test -f "{{.NODE_ROOT}}/bin/node"
     - test -f "{{.NODE_ROOT}}/bin/npm"
 
+  install-v8-build-tools:
+    desc: install gclient and gn for build v8
+    vars:
+      DEPOT_TOOLS_ROOT: '{{.BUILD_ROOT}}/depot_tools'
+      GN_INSTALL_DIR: '{{.DEPOT_TOOLS_ROOT}}/installed-gn'
+    cmds:
+    - mkdir -p '{{.DEPOT_TOOLS_ROOT}}'
+    - curl -L https://storage.googleapis.com/vectorizedio-public/dependencies/depot_tools.tar.gz | tar -C '{{.BUILD_ROOT}}/' -xz
```

Comment:
for OSS, it might make sense to use the google URLS

Reply:
Is it right that I need to install gclient in install-dependecies.sh by using google URLS.
Or do you want smth else?

Reply:
Ivo said that I can not use task from private repo in deps in rp:build. So I move it to open-source task.

Reply:
i don't understand this response tbh. 

I'm saying, why not do something like:

```
$ git clone https://chromium.googlesource.com/chromium/tools/depot_tools.git
```

or download the .tar.gz from chromium? I'm confused. The oss doesn't need to depend on our depot_tools version.

Reply:
another alternative is to have this task install from the google repo, and we can create a new `v-rp:build` internal task that installs from GCS. Similarly to how we do it with other subprojects

Reply:
left a comment on https://github.com/vectorizedio/vtools/pull/173 on alternatives to addressing this

Reply:
yes, we should have our own in vtools that comes from the static source, oss can use upstream chromium.